README.md

```diff
@@ -2,61 +2,337 @@
+```bash
+https://raw.githubusercontent.com/EBI-Metagenomics/mettannotator/master/tests/test.csv
+
+nextflow run ebi-metagenomics/mettannotator \
+   -profile <docker/singularity/...> \
+   --input test.csv \
+   --outdir <OUTDIR> \
+   --dbs <PATH/TO/WHERE/DBS/WILL/BE/SAVED>
+```
+
+<a name="out"></a>
+## Outputs
+
+The output folder structure will look as follows:
+```
+└─<PREFIX>
+   ├─antimicrobial_resistance
+   │  └─amrfinder_plus
+   ├─antiphage_defense
+   │  └─defense_finder
+   ├─biosynthetic_gene_clusters
+   │  ├─antismash
+   │  ├─gecco
+   │  └─sanntis
+   ├─functional_annotation
+   │  ├─dbcan
+   │  ├─eggnog_mapper
+   │  ├─interproscan
+   │  ├─merged_gff
+   │  ├─prokka
+   │  └─unifire
+   ├─mobilome
+   │  └─crisprcas_finder
+   ├─quast
+   │  └─<PREFIX>
+   │      ├─basic_stats
+   │      └─icarus_viewers
+   ├─rnas
+   │  ├─ncrna
+   │  └─trna
+   ├─multiqc
+   │  ├─multiqc_data
+   │  └─multiqc_plots
+   │      ├─pdf
+   │      ├─png
+   │      └─svg
+   ├─pipeline_info
+   │  ├─software_versions.yml
+   │  ├─execution_report_<timestamp>.txt
+   │  ├─execution_report_<timestamp>.html
+   │  ├─execution_timeline_<timestamp>.txt
+   │  ├─execution_timeline_<timestamp>.html
+   │  ├─execution_trace_<timestamp>.txt
+   │  ├─execution_trace_<timestamp>.html
+   │  └─pipeline_dag_<timestamp>.html
+
+```
+
+### Merged GFF
+
+The two main output files for each genome are located in `<OUTDIR>/<PREFIX>/functional_annotation/merged_gff/`:
+
+- `<PREFIX>_annotations.gff`: annotations produced by all tools merged into a single file
+
+- `<PREFIX>_annotations_with_descriptions.gff`: a version of the GFF file above that includes descriptions of all InterPro terms to make the annotations human-readable.
+
+Both files include the genome sequence in the FASTA format at the bottom of the file.
+
+#### Data sources
+Below is an explanation of how each field in column 3 and 9 of the final GFF file is populated. In most cases, information is taken as is from the reporting tool's output.
+
+| Feature (column 3)    | Attribute Name (column 9)                                               | Reporting Tool  | Description                                                                                                                                                                                                 |
+|-----------------------|-------------------------------------------------------------------------|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ncRNA                 | all                                                                     | cmscan + Rfam   | ncRNA annotation (excluding tRNA)                                                                                                                                                                           |
```

Comment:
Does 'all' indicate that the entire row in the GFF is populated by this tool?

If that is so, maybe "as is" or similar? Or all* and have an explanation in the table footer?

Reply:
It means all attributes in column 9...